tests/test_logger_block.py

```diff
@@ -97,3 +97,11 @@ def test_log_sorting(self):
             call(json.dumps(signal2.to_dict(), sort_keys=True)),
         ])
         self.assertEqual(blk.logger.error.call_count, 0)
+
+    def test_log_command(self):
```

Comment:
This method is showing up as a merge conflict 😢 
Other than that this PR looks good to me!